indico/migrations/versions/20250224_1832_4615aff776e0_add_user_logs.py

```diff
@@ -0,0 +1,57 @@
+"""Add user logs
+
+Revision ID: 4615aff776e0
+Revises: b212f6d17229
+Create Date: 2025-02-24 18:32:20.784460
+"""
+
+from enum import Enum
+
+import sqlalchemy as sa
+from alembic import op
+from sqlalchemy.dialects import postgresql
+
+from indico.core.db.sqlalchemy import PyIntEnum, UTCDateTime
+
+
+# revision identifiers, used by Alembic.
+revision = '4615aff776e0'
+down_revision = 'b212f6d17229'
+branch_labels = None
+depends_on = None
+
+
+class _UserLogRealm(int, Enum):
+    user = 1
```

Comment:
This does not match the UserLogRealm enum (`management = 2` is missing) in the model